client-src/elements/chromedash-wpt-eval-page.ts

```diff
@@ -0,0 +1,402 @@
+import {LitElement, TemplateResult, css, html, nothing} from 'lit';
+import {customElement, property, state} from 'lit/decorators.js';
+import {unsafeHTML} from 'lit/directives/unsafe-html.js';
+import {marked} from 'marked';
+import {SHARED_STYLES} from '../css/shared-css.js';
+import {Feature, FeatureNotFoundError} from '../js-src/cs-client.js';
+import {showToastMessage} from './utils.js';
+
+// Matches http or https, followed by wpt.fyi/results, followed by any non-whitespace and non-question mark characters.
+const WPT_RESULTS_REGEX = /(https?:\/\/wpt\.fyi\/results[^\s?]+)/g;
+
+@customElement('chromedash-wpt-eval-page')
+export class ChromedashWPTEvalPage extends LitElement {
+  static get styles() {
+    return [
+      ...SHARED_STYLES,
+      css`
+        :host {
+          display: block;
+          min-height: 100vh;
+          background: var(--sl-color-neutral-50);
```

Comment:
I don't think you need any of these colors, just use the default colors defined by site-wide CSS.